### PR TITLE
Update mixer docs to current release

### PR DESCRIPTION
**Pull request by tmarcu**
Signed-off-by: Tudor Marcu <tudor.marcu@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/mixer.rst [hunk 1]
```diff
@@ -22,6 +22,13 @@ add it with the :command:`swupd bundle-add` command as follows:
 
 Current mixing workflow
 ***********************
+Mixer by default runs *all* build commands in a container to ensure the
+correct version of the tooling is being used. This also allows custom mixes
+to automatically perform downstream format bumps when upstream releases
+a format bump. You can still run mixer natively by appending --native to
```
Comment (kevincwells) [inline source/clear-linux/guides/maintenance/mixer.rst]: It looks like elsewhere in this doc the convention is to add `*`'s around command flags to italicize them.

File: source/clear-linux/guides/maintenance/mixer.rst [hunk 2]
```diff
@@ -22,6 +22,13 @@ add it with the :command:`swupd bundle-add` command as follows:
...
+any of the commands. Note that you cannot run mixer if you are already in a
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/mixer.rst]: Elsewhere in this doc, notes are called out in a note box (example in step 10 under "Create ngninx web server .."). Consider calling out this note in the same manner, using the note directive.

**Comment by mvincerx** [comment]
@krisdale Can you add this as a remote and push your changes to it, please?

**Comment by krisdale** [comment]
@tmarcu Would you please enable commits to this pull request? Then I can commit the minor edits I requested. Thanks!  https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by krisdale** [comment]
@tmarcu Thanks for enabling. I pushed the changes I suggested, as a separate commit. Also marked the comments unresolved until reviewed - good catch ;)

**Comment by tmarcu** [comment]
Do you want me to rebase this onto master to resolve the conflicts?

**Comment by krisdale** [comment]
@tmarcu That would be great - please rebase and resolve the conflicts. As soon as that is done, we'll get the PR merged :)

**Comment by krisdale** [comment]
@mvincerx - @tmarcu confirmed this PR is ready to go. Ready to merge.

**Comment by mvincerx** [comment]
Thanks all for the diligence on this PR. .

**Pull request merged**